orchestration/dagster_orchestration/pytest.ini

```diff
@@ -3,3 +3,10 @@ console_output_style = progress
 env_files =
 	.env.test
 markers = e2e: marks tests as an end-to-end test (deselect with '-m "not e2e"')
+log_cli = True
```

Comment:
I've added log capture here for feedback during E2E tests, otherwise the test spins for quite awhile with little feedback.